### ISSUE TITLE
A display bug

### ISSUE THREAD

**Issue by franklinbill** [erickok/transdroid#542]
Dear developer
In the process of using, I found a bug in layout / display. First click "system" to enter the setting interface, and then click "export settings", and "setting successfully exported" will pop up. As shown in the screenshot, it is obvious that "g" of "setting" and "y" of "successfully" are blocked in half, and the lower part cannot be displayed. The cause of the problem may be an adaptation problem.
App version is 2.5.17, my mobile phone is Google nexus 6, screen resolution setting is 1440 * 2560, font size is the largest, Android system is 8.0.
The above is a description of the problem. The overall experience of the software is good. You can see that the design is very attentive. Thank you very much for the efforts of the developers all the time. I hope to receive a reply. 
Thank you!
![image](https://user-images.githubusercontent.com/18572115/78771768-49164000-79c3-11ea-9e90-34952fa60b2a.png)

**Comment by erickok** [comment]
Yes you are right. I use a library - quite an old one - and it doesn't scale with the OS-level font size setting. Something to fix...

**Comment by bwitt** [comment]
@erickok what's the library that's responsible for this?  maybe they've released a fix
